ui/README.md

```diff
@@ -0,0 +1,30 @@
+<p align="center">
```

Comment:
Update this readme to reflect our project

And give a link to this doc in the root readme

Ideally we have one master test command such as `mvn test` which executes both java and javascipt tests